packages/settings-library/src/settings_library/utils_r_clone.py

```diff
@@ -0,0 +1,46 @@
+import configparser
+from copy import deepcopy
+from io import StringIO
+from typing import Dict
+
+from .r_clone import RCloneSettings, S3Provider
+
+_COMMON_ENTRIES: Dict[str, str] = {
+    "type": "s3",
+    "access_key_id": "{access_key}",
+    "secret_access_key": "{secret_key}",
+    "region": "us-east-1",
```

Comment:
I am not sure if this is just a default or hardcoded: This AWS region might change at any time, for example if we for legal regions move a deployment to frankfurt (inside EU). So this should be configureable